.gitlab-ci.yml

```diff
@@ -0,0 +1,90 @@
+image: golang:1.11
+
```

Comment:
We need to delete the existing `.gitlab-ci.yaml` file

Reply:
Should be taken care of now, I rebased on top of master and deleted it.